libs/common-utilities/src/constants/contracts.ts

```diff
@@ -1,6 +1,19 @@
-import { KeychainList } from '..';
+import { KeychainList, ValidNetwork } from '..';
 
-export const CONTRACTS: KeychainList = {
+export type ContractKey =
+  | 'V3_FACTORY'
+  | 'LOOT_SINGLETON'
+  | 'SHARES_SINGLETON'
+  | 'BAAL_SINGLETON'
+  | 'GNOSIS_MULTISEND'
+  | 'TRIBUTE_MINION'
+  | 'POSTER';
+
+export type AddressKeyChain = { [key in ValidNetwork]?: string };
+// export type ContractAddressList = Record<ContractKey, KeychainList>;
+export type ContractAddressList = Record<ContractKey, AddressKeyChain>;
+
+export const CONTRACTS: ContractAddressList = {
```

Comment:
not sure why KeyChainList is not taking here, i commented it out and made the `AddressKeyChain` type.